internal/transport/transport.go

```diff
@@ -540,6 +540,9 @@ type CallHdr struct {
 	PreviousAttempts int // value of grpc-previous-rpc-attempts header to set
 
 	DoneFunc func() // called when the stream is finished
+
+	// Authority is used to override the `:authority` header.
```

Comment:
I understand this is an internal package, but I feel it makes sense to mention here that if this value is populated, it will be preferred over the `Host` field to be used as the value for the `:authority` header.